apps/api/src/app/shared/framework/idempotency.interceptor.ts

```diff
@@ -28,7 +28,7 @@ enum ReqStatusEnum {
   ERROR = 'error',
 }
 
-export const DOCS_LINK = 'https://docs.novu.co/additional-resources/idempotency';
+export const DOCS_LINK = 'https://docs.novu.co/additional-resources/idempotency?utm_campaign=inapp-interceptor';
```

Comment:
This is also machine-readable, we need to remove.
```suggestion
export const DOCS_LINK = 'https://docs.novu.co/additional-resources/idempotency';
```